third-party/chpl-venv/venv-use-sys-python.py

```diff
@@ -4,32 +4,176 @@
 
 import os
 import sys
+import re
+
+# This fixes up the chpl-venv installation in a Cray Chapel module that was built using a non-standard
+# Python installation (ie, the Python used was not installed under "/usr/...").
+#
+# 1. Remove the python executables from the virtualenv.
+#    This effectively forces the virtualenv scripts to use the system python.
+# 2. Rewrite symlinks in the virtualenv that point to the non-standard Python installation
+# 3. Rewrite shebang lines (in scripts) in the virtualenv that point to the non-standard Python exe.
+
+# Without this, "start_test" will not work if the Cray Chapel module is installed on a different
+# machine (ie, if no Python is installed in the same location as when the module was built).
+
+# As noted, these fixes are only applicable when building a Cray Chapel module under certain conditions.
+# This has been tested only for the cray-xc platform, where the Cray Chapel module was:
+#  - built on a cray-xc CLE 5 (system Python version 2.6, non-standard user-installed Python 2.7), and
+#  - installed on a cray-xc CLE 6 (system Python version 2.7).
+
+
+def rm_python_bins( bindir ):
```

Comment:
No need to [wrap arguments with spaces](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)

Should be:

```python
def rm_python_bins(bindir):
```